README.md

```diff
@@ -365,12 +365,15 @@ capturing a video clip, the `CaptureErrorCB` callback executes with a
 
 - The __limit__ property is ignored.  Only one video is recorded per invocation.
 
+- iOS supports an additional __quality__ property, to allow capturing video at different qualities.  A value of `0.5` ( the default ) means medium quality, value of `1` means high quality and value of `0` means low quality.
```

Comment:
Change here as well to say default is `1` to match with Android.

Reply:
I moved the `__quality__ property` up and kept the specific value and details with links for iOS and Android.